src/simple_cache_client.rs

```diff
@@ -1859,6 +1862,397 @@ impl SimpleCacheClient {
         self.set_difference(cache_name, set_name, elements).await
     }
 
+    /// Fetches a sorted set from a Momento Cache.
+    ///
+    /// *NOTE*: This is preview functionality and requires that you contact
+    /// Momento Support to enable these APIs for your cache.
+    ///
+    /// # Arguments
+    ///
+    /// * `cache_name` - name of cache.
+    /// * `set_name` - name of the set.
+    /// * `order` - specify ascending or descending order
+    /// * `limit` - optionally limit the number of results returned
+    ///
+    /// # Example
+    /// ```
+    /// # fn main() -> momento_test_util::DoctestResult {
+    /// # momento_test_util::doctest(|cache_name, auth_token| async move {
+    /// use std::time::Duration;
+    /// use momento::{Order, SimpleCacheClientBuilder};
```

Comment:
`momento::Order` is a pretty aggressive namespace. I think you want to scope it under something sorted set related.